mica/starcheck/tests/test_catalog_fetches.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from Quaternion import normalize, Quat
+from Chandra.Time import DateTime
+from kadi import events
+from Ska.engarchive import fetch
+from Ska.quatutil import radec2yagzag, yagzag2radec
+
+from .. import starcheck
+
+def get_cmd_quat(date):
+    date = DateTime(date)
+    cmd_quats = fetch.MSIDset(['AOCMDQT{}'.format(i) for i in [1, 2, 3]], date.secs, date.secs + 120)
+    cmd_q4 = np.sqrt(np.abs(1 - cmd_quats['AOCMDQT1'].vals[0]**2
+                            - cmd_quats['AOCMDQT2'].vals[0]**2
+                            - cmd_quats['AOCMDQT3'].vals[0]**2))
+    return Quat(normalize([cmd_quats['AOCMDQT1'].vals[0],
+                                      cmd_quats['AOCMDQT2'].vals[0],
+                                      cmd_quats['AOCMDQT3'].vals[0],
+                                      cmd_q4]))
+
+
+def get_trak_cat_from_telem(date, cmd_quat):
```

Comment:
Requiring a separate call to `get_cmd_quat` seems inconvenient in the API.  Is there a reason for breaking this out?  You can at least make the parameter be `cmd_quat=None` and then get it if None.  But agreed it might be useful to maintain `get_cmd_quat` as a separate utility function in chandra_aca.